docs/site/Application-setup-tips.md

```diff
@@ -0,0 +1,80 @@
+---
+lang: en
+title: 'Tips for application setup'
```

Comment:
I believe we're trying to use the title style where the first character of each word being upper case (though there're some inconsistencies in the docs).

```suggestion
title: 'Tips for Application Setup'
```